com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbePlacement.cs.hlsl

```diff
@@ -0,0 +1,52 @@
+//
+// This file was automatically generated. Please don't edit by hand. Execute Editor command [ Edit > Rendering > Generate Shader Includes ] instead
+//
+
+#ifndef PROBEPLACEMENT_CS_HLSL
+#define PROBEPLACEMENT_CS_HLSL
+// Generated from UnityEngine.Experimental.Rendering.ProbePlacement+GPUProbeVolumeOBB
+// PackingRules = Exact
+struct GPUProbeVolumeOBB
+{
+    float3 corner;
+    float3 X;
+    float3 Y;
+    float3 Z;
+    int minSubdivisionLevel;
+    int maxSubdivisionLevel;
+    float geometryDistanceOffset;
+};
+
+//
+// Accessors for UnityEngine.Experimental.Rendering.ProbePlacement+GPUProbeVolumeOBB
+//
+float3 GetCorner(GPUProbeVolumeOBB value)
```

Comment:
Do we need these accessors? I don't see them being used?

Reply:
Yup, there is a needAccessor in the attribute GenerateHLSL

Reply:
Thanks